drivers/include/fxos8700.h

```diff
@@ -47,8 +47,8 @@ extern "C" {
  * If set to 0, measurements will be converted to mg.
  * If set to 1, raw adc readings will be returned.
  */
-#ifndef FXOS8700_USE_ACC_RAW_VALUES
-#define FXOS8700_USE_ACC_RAW_VALUES (0)
+#ifndef CONFIG_FXOS8700_USE_ACC_RAW_VALUES
+#define CONFIG_FXOS8700_USE_ACC_RAW_VALUES (0)
```

Comment:
Let's try to make this have the same behaviour as the Kconfig boolean:
```suggestion
#ifdef DOXYGEN
#define CONFIG_FXOS8700_USE_ACC_RAW_VALUES
```
And also you can replace the usages in the rest of the code with C conditionals and `IS_ACTIVE`

Reply:
This however does not model as boolen as CONFIG_FXOS8700_USE_ACC_RAW_VALUES needs to be defined with 0 or 1 in all cases.